### PR TITLE
Implement storage upload queue, retry count and request timeout

### PR DESCRIPTION
**Pull request by Koochr**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,6 +13,9 @@
     "@openzeppelin/contracts": "^4.4.2",
     "@solana/web3.js": "^1.29.2",
     "@truffle/hdwallet-provider": "^1.5.0",
+    "@types/config": "^0.0.41",
```
Comment (Koochr) [inline package.json:16]: IDK why this changes appeared, they should be in develop already as they were added with typescript migration

Comment (Koochr) [reply package.json:16]: I'm not sure. We have typescript in main dependencies, why should types go to dev dependencies?
Also, I know approach, when docker is not installing dev dependencies, but it's also not our case.

Comment (tankakatan) [reply package.json:16]: Please move typescript to `devDependencies`. `dependencies` should only contain packages that are used in the app's runtime.

File: src/db/models/index.ts [hunk 2]
```diff
@@ -30,7 +30,7 @@ class SequelizeFactory implements SequelizeFactory {
                 transactionType: this.config.transactionType,
                 retry: {max: this.config.retry.max},
                 logQueryParameters: true,
-                logging: log.debug.bind(log)
+                logging: config.get('db.logging') ? log.debug.bind(log) : false
```
Comment (Koochr) [inline src/db/models/index.ts:33]: I've added this, bc now db floods console with debug messages. Default behavior is not changed, but now you can override it by local config file

File: src/util/migrate.ts [hunk 3]
```diff
@@ -27,20 +29,21 @@ export default (async () => {
...
         storage: new SequelizeStorage({sequelize}),
-        logger: logger.child({module: 'migrate'})
+        logger: log
     });
 
     try {
         await umzug.up();
         await sequelize.close();
     } catch (e) {
+        log.error(e);
```
Comment (Koochr) [inline src/util/migrate.ts:46]: It was super ambiguous without this log when migration failed. We should notify user here.

File: src/db/models/index.ts (hunk 2 above)
Comment (tankakatan) [reply src/db/models/index.ts:33]: This makes sense, but please be more specific introducing new config options. This particular one only controls the database logging, right? Please rename it appropriately. And also set it to `false` in `production.yaml`.

**Review by tankakatan — APPROVED** [review]
